implementation/src/main/java/io/smallrye/jwt/config/JWTAuthContextInfoProvider.java

```diff
@@ -135,6 +157,16 @@ private static JWTAuthContextInfoProvider create(String publicKey, String public
     @Inject
     @ConfigProperty(name = "smallrye.jwt.groups.path")
     private Optional<String> groupsPath;
+    @Inject
+    @ConfigProperty(name = "smallrye.jwt.expiration.grace", defaultValue = "60")
+    private Optional<Integer> expGracePeriodSecs;
+    /**
+     * List of algorithms to whitelist JWT validation based on jose4j algorithms
+     * list org.jose4j.jws.AlgorithmIdentifiers.
+     */
+    @Inject
+    @ConfigProperty(name = "smallrye.jwt.whitelist.algorithms")
+    private List<String> whitelistAlgorithms;
```

Comment:
This is what was causing a failure - the property needs to be Optional, please update

Reply:
Done :)